### ISSUE TITLE
Opening file from outside default folder causes Kode to try to open two more files.

### ISSUE THREAD

**Issue by RobsterCraw** [TN-1/Kode#39]
Using the no-install package, I tried opening a file in a different KSP folder (as usual, in /ships/Scripts/).  On doing so, two extra open file windows pop up.  The problem didn't occur when opening files in the default KSP/Ships/Scripts/ folder.

**Comment by TN-1** [comment]
I just tried it out as you described and nothing happened.  
To be clear, Default KSP is the currently selected KSP install(In the toolbar)? And a different KSP folder is an install that you navigated to in the "Open File" dialog?

**Comment by RobsterCraw** [comment]
Yes that is correct.  I want to say that this didn't happen when I was using the win-installer version on my older machine (I am using the no-installer version on a new computer), but I don't remember trying this exact procedure.
